hatchery/src/lib.rs

```diff
@@ -9,11 +9,14 @@ mod error;
 mod instance;
 mod memory;
 mod snapshot;
+mod snapshot_bag;
 mod storage_helpers;
 mod world;
+mod world_snapshot;
 
 pub use error::Error;
 pub use world::{Event, Receipt, World};
+pub use world_snapshot::WorldSnapshotId;
```

Comment:
I suggest switching it around, and do the following renames:
```text
`SnapshotId` -> `ModuleSnapshotId`
`WorldSnapshotId` -> `SnapshotId`
```

Reply:
good suggestion, will do that